Sources/Testing/Testing.docc/MigratingFromXCTest.md

```diff
@@ -624,18 +621,18 @@ Additional options can be specified when calling `XCTExpectFailure()`:
 
 - [`isEnabled`](https://developer.apple.com/documentation/xctest/xctexpectedfailure/options/3726085-isenabled)
   can be set to `false` to skip known-issue matching (for instance, if a
-  particular issue only occurs under certain conditions); and
+  particular issue only occurs under certain conditions)
 - [`issueMatcher`](https://developer.apple.com/documentation/xctest/xctexpectedfailure/options/3726086-issuematcher)
   can be set to a closure to allow marking only certain issues as known and to
-  allow other issues to be recorded as test failures.
+  allow other issues to be recorded as test failures
 
 The testing library includes overloads of `withKnownIssue()` that take
 additional arguments with similar behavior:
 
 - ``withKnownIssue(_:isIntermittent:fileID:filePath:line:column:_:when:matching:)-68e5g``
 - ``withKnownIssue(_:isIntermittent:fileID:filePath:line:column:_:when:matching:)-7azqg``
 
-To conditionally enable known-issue matching and/or to match only certain kinds
+To conditionally enable known-issue matching or to match only certain kinds
```

Comment:
We don't want it to read like you have to choose between them and can't have both. Is there a better way to say "and/or"?

Reply:
Similar to above, the general preference is to not use "and/or", and to use one of the conjunctions, or to do something like, "To either conditionally enable known-issue matching, or match only certain kinds
of issues, or both:", which felt kind of awkward.

Thoughts?

Reply:
I'm fine going with whatever the style guide says, just didn't want to inadvertently make it read like either-or.